### PR TITLE
Render stacked charts.

### PR DESCRIPTION
**Pull request by saipramod**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,16 +1,77 @@
...
+                        />
+                        <Dropdown
+                            dropdownWidth={100}
+                            styles={{ dropdown: { width: 100 } }}
+                            selectedKey={this.state.contact}
+                            options={contactOptions}
+                            label="Social Distancing"
+                            onChange={(e, item) =>
+                                this.setContact(item?.key as string)
+                            }
```
Comment (saipramod) [inline src/App.tsx:57]: I moved up the controls to select `contact` and `type` to `App`. The control donot render so well. If could please take a look at this I would be thankful. After the PR's are merged i will work on moving to an `arearange` or something similar.

File: src/service.ts [hunk 2]
```diff
@@ -22,3 +23,17 @@ export const getDates = async () => {
         alert(err);
     }
 };
+
+export const getStackedChart = async (options: {
+    contact: string;
+    type: string;
+}): Promise<StackedChartData[] | undefined> => {
+    try {
+        const response = await axios.get(`${backendUrl}/stackedchart`, {
```
Comment (lzl124631x) [inline src/service.ts:32]: use hyphen between words in url https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris

File: src/stackedchart/StackedChart.sass [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.chart-container
+  margin-right: 5px
```
Comment (lzl124631x) [inline src/stackedchart/StackedChart.sass:2]: I'll use variables later

File: src/stackedchart/StackedChartData.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
+export interface StackedChartData{
+    title: string,
```
Comment (lzl124631x) [inline src/stackedchart/StackedChartData.ts:2]: looks like we still have some charting related field names. I can help change that later.

File: src/service.ts (hunk 2 above)
Comment (saipramod) [reply src/service.ts:32]: will change this in my next PR

**Pull request merged**